__tests__/attributes.test.js

```diff
@@ -7,7 +7,7 @@ describe('attributes', () => {
 		expect(instance.vendorID).toBe('foo');
 		expect(instance.apiKey).toBe('bar');
 		expect(instance.publicKey).toBe('MISSING');
-		expect(instance.server).toBe('https://vendors.paddle.com/api/2.0');
+		expect(instance._getServer()).toBe('https://vendors.paddle.com/api/2.0');
```

Comment:
These will need to be updated as well.

Reply:
Thanks for these comments! I'll try to resolve them soon'ish.